ed_sensor_integration/src/kinect/fitter.cpp

```diff
@@ -458,6 +457,19 @@ void Fitter::checkExpectedBeamThroughEntity(const std::vector<double>& model_ran
 
 // ----------------------------------------------------------------------------------------------------
 
+void Fitter::configureBeamModel(const image_geometry::PinholeCameraModel& caminfo)
+{
+    uint nr_beams = std::min(200, caminfo.fullResolution().width); // don't use more data points than the resolution of your camera
+    double fx = caminfo.fx();
+    double fx_resize = fx * nr_beams / caminfo.fullResolution().width; // Reducing nr of data points will require a different focal length
+    double w = 2 * double(nr_beams) / fx_resize; // reverse calculation of the width of the beam model.
```

Comment:
int divided by a double will result in a double.
```suggestion
    double w = 2 * nr_beams / fx_resize; // reverse calculation of the width of the beam model.
```